### PR TITLE
Fix conflict message when referenced snippet is being deleted

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR fixes the message when a referenced snippet is tried to be deleted.

#### Why?

Because now a JS error was thrown, which caused all other clicks to have no effect.

#### To Do

- [ ] Check `SnippetController::putAction` with referential integrity message

### CODE REVIEW

File: src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php [hunk 1]
```diff
@@ -117,7 +123,8 @@ public function __construct(
         FormFactory $formFactory,
         RequestHashChecker $requestHashChecker,
         ListRestHelper $listRestHelper,
-        MetadataFactoryInterface $metadataFactory
+        MetadataFactoryInterface $metadataFactory,
+        TranslatorInterface $translator
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php]: Should we document BC Breaks in constructors/services definitions @sulu/core-team?

**Comment by alexander-schranz** [comment]
@danrot Would call the confirmation button something like `Delete` instead of Ok.

Bildschirmfoto 2020-01-20 um 10 31 06

@chirimoya what do you think?

File: src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php (hunk 1 above)
Comment (danrot) [reply src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php]: That's the reason I would not do it. It is very unlikely that this is done, and if we start to document BC breaks on that level, then the UPGRADE.md file gets very bloated.

**Comment by danrot** [comment]
@alexander-schranz The button was already there before, haven't touched it :see_no_evil: And if we rename it here, it should probably also be renamed in the previous overlay, where it currently also says "Ok".

**Pull request merged**